python/dolfinx/io/gmshio.py

```diff
@@ -264,10 +290,16 @@ def model_to_mesh(
         num_nodes = cell_information[perm_sort[-1]]["num_nodes"]
         cell_id, num_nodes = comm.bcast([cell_id, num_nodes], root=rank)
 
-        # Check for facet data and broadcast relevant info if True
+        # Check for facet, edge and vertex data and broadcast relevant info if True
         has_facet_data = False
         if tdim - 1 in cell_dimensions:
```

Comment:
I know this is existing code that are been replicated below, but it would be better as something like
```python
has_facet_data = (tdim - 1) in cell_dimensions
```

Reply:
I think this is better

Reply:
Fixed in https://github.com/FEniCS/dolfinx/pull/3547/commits/0a875880936454e9140c27e051479c30ab335062